app/mobile-content/exhibits/edit-exhibit-pages/edit-exhibit-pages.component.ts

```diff
@@ -19,78 +19,85 @@ import { Status } from '../../shared/status.model';
 })
 export class EditExhibitPagesComponent implements OnInit {
   @Input() exhibitId: number;
-  pages: ExhibitPage[];
+  infoPages = new Map<number, MobilePage>();
+  pages: MobilePage[];
   searchStatusOptions = Status.getValuesForSearch();
-  selectedStatus = 'ALL';
+  selectedStatus: statusTypeForSearch = 'ALL';
   statusOptions = Status.getValues();
 
-  private createDialogRef: MdDialogRef<CreateExhibitPageDialogComponent>;
   private deleteDialogRef: MdDialogRef<ConfirmDeleteDialogComponent>;
+  private editDialogRef: MdDialogRef<EditPageDialogComponent>;
+  private selectDialogRef: MdDialogRef<SelectPageDialogComponent>;
 
   constructor(private dialog: MdDialog,
-              private exhibitPageService: ExhibitPageService,
               private exhibitService: ExhibitService,
+              private pageService: MobilePageService,
               private toasterService: ToasterService,
               private translateService: TranslateService) {}
 
   ngOnInit() {
     this.reloadList();
   }
 
-  createPage() {
-    this.createDialogRef = this.dialog.open(CreateExhibitPageDialogComponent, { width: '75em' });
-    this.createDialogRef.afterClosed().subscribe(
-      (newPage: ExhibitPage) => {
-        if (newPage) {
-          newPage.exhibitId = this.exhibitId;
-          this.exhibitPageService.createPage(newPage)
-            .then(
-              newId => {
-                newPage.id = newId;
-                this.pages.push(newPage);
-                this.toasterService.pop('success', this.translateService.instant('page created'));
-              }
-            ).catch(
-              () => this.toasterService.pop('error', this.translateService.instant('create failed'))
-            );
-        }
+  addPage() {
+    this.selectDialogRef = this.dialog.open(SelectPageDialogComponent, { width: '75%' });
```

Comment:
Why not? em units relate to font size, while % (in this case) relate to screen size. I thought it fits better.